backend/ttrs/serializers.py

```diff
@@ -5,14 +5,22 @@
 from rest_framework.exceptions import ValidationError
 from django.core.exceptions import ValidationError as DjangoValidationError
 
-from .models import Student, College, Department, Major, Course, Lecture, TimeTable
+from .models import Student, College, Department, Major, Course, Lecture, Evaluation, TimeTable
 
 
 class StudentSerializer(serializers.ModelSerializer):
```

Comment:
I reordered Meta class and several methods, in order of Meta class, get field methods and validation methods.